src/pipecat/services/cartesia.py

```diff
@@ -173,6 +174,12 @@ async def _receive_task_handler(self):
                         num_channels=1
                     )
                     await self.push_frame(frame)
+                elif msg["type"] == "error":
+                    logger.error(f"Cartesia error: {msg}")
+                    await self.stop_all_metrics()
+                    await self.push_frame(ErrorFrame(f'Cartesia error: {msg["error"]}'))
```

Comment:
Nit: can we also include the processor name in both strings? `{self}` should do it. This way it's easier to identify which processor it is (in case there are multiple cartesia ones).